README.md

```diff
@@ -9,46 +9,58 @@ Dostupné na [naucse.python.cz](https://naucse.python.cz).
 ## Instalace a spuštění
 
 Chceš-li server spustit na svém počítači, např. proto, že se chceš zapojit
-nebo abys ho měl/a k dispozici i bez připojení k Intenetu, je potřeba ho
-nejdřív nainstalovat:
+do vývoje, je potřeba ho nejdřív nainstalovat:
 
-* Vytvoř a aktivuj si [virtuální prostředí](https://naucse.python.cz/lessons/beginners/install/) v Pythonu 3.6.
+* (nepovinné) Vytvoř a aktivuj si [virtuální prostředí](https://naucse.python.cz/lessons/beginners/install/) v Pythonu 3.6.
 * Přepni se do adresáře s kódem projektu.
 * Nainstaluj závislosti:
-   ```console
-   $ python -m pip install -r requirements.txt
-   ```
 
-Nainstalovanou aplikaci spustíš následovně:
+  * Linux/Mac:
 
-* Aktivuj si virtuální prostředí.
-* Nastav proměnnou prostředí:
-  * Linux a Mac OS:
     ```console
-    $ export PYTHONPATH=.
+    $ python3 -m pip install pipenv
+    $ pipenv install
     ```
+
   * Windows:
-    ```console
-    > set PYTHONPATH=.
+
+    ```doscon
+    > py -3 -m pip install pipenv
```

Comment:
tohle funguje, když máš aktivovaný venv?

Reply:
Teď jsem to zkoušel u sebe (Windows 10, Python 3.6). Pipenv to nainstaluje globálně, ale pipenv pak detekuje jestli máš aktivovaný virtualenv a instaluje správně do virtualenvu.